tests/tests_pytorch/strategies/test_fsdp.py

```diff
@@ -221,19 +261,38 @@ def custom_auto_wrap_policy(
 
 @RunIf(min_cuda_gpus=2, skip_windows=True, standalone=True, min_torch="1.12")
 @pytest.mark.parametrize(
-    "model, strategy",
+    "model, strategy, strategy_cfg",
     [
-        (TestFSDPModel(), "fsdp"),
-        (TestFSDPModelAutoWrapped(), FSDPStrategy),
+        (TestFSDPModel(), "fsdp", None),
+        pytest.param(
+            TestFSDPModelAutoWrapped(),
+            FSDPStrategy,
+            {"auto_wrap_policy": custom_auto_wrap_policy},
+            marks=RunIf(max_torch="2.0.0"),
+        ),
+        pytest.param(
+            TestFSDPModelAutoWrapped(),
+            FSDPStrategy,
+            {"auto_wrap_policy": custom_auto_wrap_policy_2x},
+            marks=RunIf(min_torch="2.0.0"),
+        ),
+        pytest.param(
+            TestFSDPModelAutoWrapped(),
+            FSDPStrategy,
+            {"auto_wrap_policy": custom_fsdp_policy, "use_orig_params": True},
+            marks=RunIf(min_torch="2.0.0"),
+        ),
     ],
+    ids=["manually_wrapped", "autowrap_1x", "autowrap_2x", "autowrap_use_orig_params"],
```

Comment:
Minor nit: if specifying ids, then maybe let's move them to the pytest.param() above so it's easier to read/track.